src/components/github-htmlview.component.js

```diff
@@ -294,6 +303,32 @@ export class GithubHtmlView extends Component {
 
           return undefined;
         },
+        details: (node, index, siblings, parent, defaultRenderer) => {
+          const tags = onlyTagsChildren(node);
```

Comment:
Nice nice nice 🎉 
Let's check that there are indeed two tags and that they are indeed summary and hidden.
Define them here with consts for later use

Reply:
hmm, I'm thinking how we should handle the case of no `summary` or no `hidden` tags in `details` tag.
May need a default `summary` and empty `hidden` for this case.

Reply:
This shouldn't really happen I think, but I'm ok with your suggestion
The goal is to avoid a crash on [1]

Reply:
What `[1]` means?